tests/integration/scripteditor.ts

```diff
@@ -89,7 +89,10 @@ describe('Script Editor tests', () => {
     cy.createNewScriptEditor('Python');
 
     // Add some text to the editor
-    cy.get('span[role="presentation"]').type('print("test")\n');
+    cy.get('span[role="presentation"]').type('print("test")');
+
+    cy.wait(1000);
```

Comment:
is a full second necessary or would 500 ms do instead? I'm just worried about adding more waits.

Reply:
It was arbitrary, I can try 500